### DISCUSSION TITLE
"export 'ClassNames' was not found in 'react-day-picker'Bug:

### DISCUSSION THREAD

**Discussion by Zeb1986** [gpbl/react-day-picker#1873] [Support]
Trying to make everything like in docs. 
import { ClassNames, DayPicker } from 'react-day-picker';
import styles from 'react-day-picker/dist/style.module.css';
but getting this error on build

**Comment by gpbl** [comment]
@Zeb1986 please provide more details about your error on build, as it seems an issue on your side. Here an example using `ClassNames`:

https://codesandbox.io/s/tender-gagarin-234mts?file=/src/App.tsx
